src/iterators/mod.rs

```diff
@@ -26,7 +27,7 @@ pub use self::chunks::{ExactChunks, ExactChunksIter, ExactChunksIterMut, ExactCh
 pub use self::lanes::{Lanes, LanesMut};
 pub use self::windows::Windows;
 
-use std::slice::{self, Iter as SliceIter, IterMut as SliceIterMut};
+use alloc::slice::{self, Iter as SliceIter, IterMut as SliceIterMut};
```

Comment:
This could still be std (it's in core)

Reply:
Thanks for reviewing! I will modify it in the next commit